src/pages/ProductDetail/Overview/SelectOption/Cart/Product/Product.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import ProductOptions from './ProductOptions/ProductOptions';
+import './Product.scss';
+
+class Product extends Component {
+  render() {
+    const { product_name, seller, options } = this.props.product;
+    console.log(options);
+    return (
+      <div className='Product'>
+        <div className='productTitle flexBox'>{seller}</div>
+        <div className='productCheckBox flexBox'>
+          <input type='checkbox' classname='product' id='product' />
+          <label for='product'></label>
+        </div>
+        <div className='deleteBox flexBox'>X</div>
+        <div className='product'>
+          <div className='productOptionBox'>
+            <div className='productImageBox'>
+              <img
+                src='/images/post1.png'
+                className='ProductImage'
+                alt='productImage'
+              />
+            </div>
+            <div className='flexTitle'>
+              <div className='title'>{product_name}</div>
+              <div className='titleOption'>판매자직접배송 | 무료배송</div>
+            </div>
+          </div>
+          {options.length &&
+            options.map((option) => <ProductOptions option={option} />)}
+        </div>
+        <div className='flexTotalBox'>
+          <div className='optionChange'>옵션 변경 | 바로 구매</div>
+          <div className='totalCalculation'>
+            {options.length &&
+              options
+                .reduce(
+                  (accumulator, currentValue) =>
+                    accumulator +
+                    parseInt(currentValue.value * currentValue.count),
+                  0
+                )
+                .toString()
+                .replace(/\B(?=(\d{3})+(?!\d))/g, ',') + '원'}
```

Comment:
아 이부분도 사용되는 함수군요 그러면 컴포넌트로 빼는게 좋겠네요!

```suggestion 

// 함수만 분리하고
export default function getTotalValue(targetList) {
  if (targetList.length) {
    return (
      targetList
        .reduce(
          (accumulator, currentValue) =>
            accumulator + parseInt(currentValue.value * currentValue.count),
          0
        )
        .toString()
        .replace(/\B(?=(\d{3})+(?!\d))/g, ",") + "원"
    );
  }
}

// 이런식으로 import해서 사용
   <div className="totalCalculation">{getTotalValue(options)}</div>
                    
                                        
               
```

Reply:
결재완료!